chopper_tune.cfg

```diff
@@ -199,14 +204,27 @@ gcode:
             {% set minZAX = minAX %}
             {% set acceleration = settings.printer.max_z_accel|int %}
             {% set trv_speed = config.printer.max_z_velocity|int / 2 * 60 %} ; Idle movements speed
-        {% else %}
-            {action_raise_error('WARNING!!! Incorrect direction')}
+        {% elif axis != 'e'%}
+            {action_raise_error('WARNING!!! Incorrect direction', axis)}
         {% endif %}
 
     {% else %}
         {action_raise_error('WARNING!!! Script does not support your kinematics')}
     {% endif %}
 
+    {% if axis == "e" %}
+        {% set axes = ['e', 'z'] %}
+        {% set steppers = ['extruder'] %}
+        {% set minAX = 0 %}
+        {% set maxAX = 100 %}
+        {% set midAX = maxAX / 2 %}
+        {% set midBAX = 50 %}
+        {% set minZAX = 50 %}
```

Comment:
These constants are arbitrary and may not be suitable for some users. By the way, `midAX` you could probably specify 0 so that the extruder returns back to its original position, and don't go up to 100 / 2

Reply:
Absolutely, but this has to be defined and has no importance if the extruder is not extruding some filament... It could be run indefinitely in the same direction...